recipes/tailwind/templates/config/tailwind.config.js

```diff
@@ -1,6 +1,8 @@
 // tailwind.config.js
+const {paths} = require("blitz/installer")
+
 module.exports = {
-  content: ["{pages,app}/**/*.{js,ts,jsx,tsx}"],
+  content: [`${paths.appSrcDirectory({config: true})}/**/*.{js,ts,jsx,tsx}`],
```

Comment:
I wonder if this is a performant solution. It does mean, that on every run TW needs to first to the `fs.existsSync`, doesn't it?

I suggest adding the two possible places and let do the check for files. 

I don't know how the Next 13 app+pages-structure will work but assume that the Next 13 app directory needs to go into the blitz-src-folder as well, right? So there is no clash to be expected there.

```diff
-   content: [`${paths.appSrcDirectory({config: true})}/**/*.{js,ts,jsx,tsx}`],
+   content: ["{src,app}/**/*.{js,ts,jsx,tsx}"],
```

This would also simplify the lookup by removing the config-option.

---

About that, what does this `return "{pages,app}"` – the `pages, app` – actually stand for? The TW config is in root, so there is no "pages" folder on root, right? I never understood why the default config puts it like that. I changed it to just "`./src/*…`" for my setup and that works fine. Hence my change to have this part `{src,app}` above.

Reply:
@tordans great advice!

So to answer your question, I believe the earlier versions of blitz had a pages directory in the root together with app directory hence why we used `{pages,app}`.

@dillonraphael  I'm not sure if we still need to support `{app,pages,src}` or if we can go with `{app,src}` this time as adding something like `{pages,app,src}` may result in a "No such file or directory" error for recent projects.

Reply:
There is still a chance the user may have the `pages` directory in their root. For example if they are using a default nextjs setup, most likely they will have their `pages` directory in the root and not in `src`.

Reply:
Okay, so from the above explanations I believe it won't error if I put all the directories in to consideration ie. `{app,pages,src}` and we can get rid of the `isConfig` param.

Reply:
> Okay, so from the above explanations I believe it won't error if I put all the directories in to consideration ie. {app,pages,src} and we can get rid of the isConfig param.

I tested and confirmed it in my blitz app just now, having `{app,pages,src}` but only a `src` directory is no issue.

If we where to follow the https://tailwindcss.com/docs/guides/nextjs recommendations to the letter, this would become:

```
  content: [
    "./src/**/*.{js,ts,jsx,tsx}",
    "./app/**/*.{js,ts,jsx,tsx}",
    "./pages/**/*.{js,ts,jsx,tsx}",
  ],
```

(They spread it, but they also have a `./` at the start.)

Reply:
Thanks, @tordans I have updated the code to have `./{pages,src,app}/**/` I've tested and this seems to work.